alerts/alerts.js

```diff
@@ -63,6 +63,20 @@ const formatMessage = (snsMessage, colorCode, snsMessageFooter) => {
   }
 };
 
+const buildMessageRequest = (snsMessage, colorCode, snsMessageFooter) => {
+  const body = formatMessage(snsMessage, colorCode, snsMessageFooter);
+  if (process.env.DEPLOY_ENVIRONMENT === "integration") {
+    body.channel = process.env.SLACK_CHANNEL_ID;
```

Comment:
how does the prod channel get set?

Reply:
this was also going to be my question 😆

Reply:
I could be wrong but from a light prodding I think Slack allows a default channel to be set but this can be overridden if a channel is set in the body of the fetch call. Which is why a channel wasn't specified before but because we want to send integration notifications to a different channel, by including it in the message body we can direct them to our specified channel

@ethanmills have I understood this correctly?

Reply:
Yeah, that